components/PostContent/PostContent.tsx

```diff
@@ -15,53 +15,54 @@ export default function PostContent( props: {
 			{
 				props.blocks.map( ( block, i ) => {
 					const blockProps = mapAttributesToProps( block.attributes || [] );
-					const key = `block-${i}`;
+					const defaultProps = { key: `block-${i}` };
 
 					switch ( block.name ) {
 						case 'core/image':
+							const imageProps = {
+								...defaultProps,
+								src: blockProps.src,
+								srcSet: blockProps.srcset,
+								key: blockProps.key,
```

Comment:
`key` should never be overridden from WordPress data. It's purely a React construct.
```suggestion
```

Reply:
@chriszarate Good catch. I miscopied it and just fixed. Sorry